### ISSUE TITLE
a11y - <code> color does not meet contrast ratio of 4.5

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#630]
## Problem
Current ratio is 3.8 for small text which gets flagged.

## Proposed solution
Increase current ratio specific to the hawkid login link.
